src/tests/utils/helper.rs

```diff
@@ -59,14 +39,11 @@ pub fn generate_random_string() -> String {
     (0..10).map(|_| rng.gen_range('a'..='z')).collect()
 }
 
-pub fn mock_data() -> Result<(KeyManager, PublicKey, MuSig2Signer), anyhow::Error> {
+pub fn mock_data() -> Result<(KeyManager, PublicKey), anyhow::Error> {
     let path = format!("test_output/{}", generate_random_string());
-    let config = StorageConfig::new(path, None);
-    let store = Rc::new(Storage::new(&config)?);
-    let ket_manager_key = format!("test_output/{}", generate_random_string());
-    let key_manager = create_key_manager(ket_manager_key.as_str(), store.clone())?;
-    let pub_key = create_pub_key(&key_manager)?;
-    let musig = MuSig2Signer::new(store.clone());
+    let password = "secret password_123__ABC".to_string();
```

Comment:
@TomasSzeSirk @jonasmartin this might do the trick, but i dont fully understand the rational of this password update